### DISCUSSION TITLE
Option to migrate Settings from old SD Maid

### DISCUSSION THREAD

**Discussion by elhennig** [d4rken-org/sdmaid-se#479] [Q&A]
Will there be an option to migrate the configuration from the old SD Maid app to the new one?

**Comment by d4rken** [comment]
I plan to add migration for custom system cleaner filters and for exclusions likely too.

But not for general settings.

Settings in new SD Maid are not 1:1 the same and might behave slightly different, so users should reevaluate what settings they use and re familiarize themselves with the configuration.
